sample-input/pin-cell/PC-isotope.py

```diff
@@ -0,0 +1,274 @@
+from openmoc import *
+import openmoc.log as log
+import openmoc.plotter as plotter
+from openmoc.options import Options
+import h5py as h5
+import numpy as np
+
+###############################################################################
+#######################   Main Simulation Parameters   ########################
+###############################################################################
+
+options = Options()
+
+num_threads = options.getNumThreads()
+track_spacing = options.getTrackSpacing()
+num_azim = options.getNumAzimAngles()
+tolerance = options.getTolerance()
+max_iters = options.getMaxIterations()
+
+log.set_log_level('NORMAL')
+
+
+###############################################################################
+###########################   Creating Isotopes    ############################
+###############################################################################
+
+log.py_printf('NORMAL', 'Importing isotope data from HDF5...')
+
+mgxs = h5.File('mgxs-8-groups.h5')
+mgxs = mgxs['material']
+
+# O-16 in Fuel
+o16f = Isotope(name='O-16 in Fuel')
+o16f.setNumEnergyGroups(8)
+xs = mgxs['material 10000']['O-16']
```

Comment:
You should use a stripped down version of the HDF5 data file. In particular, rename the 'material 10000' -> '1' or something simpler. In addition, remove the 'average' and 'std. dev.' groups in place of a single group for each xs.